django/db/migrations/loader.py

```diff
@@ -6,11 +6,15 @@
 
 from django.apps import apps
 from django.conf import settings
+from django.db import connection as connection_
```

Comment:
It's a little delicate... neither `makemigrations` nor `migrate` should work at this state, but I think `showmigrations` should work and this breaks it as well.